TheOracle2/Interactions/SlashCommands/PlayerCharacterCommandGroup.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Discord.Interactions;
```

Comment:
the `/player-character` group is intended for "meta" operations: creating, deleting, setting attribute values (rather than incrementing/decrementing), etc. since typically they're used relatively infrequently (once a session at most), it's not a big deal if they're 'gated' behind a command group.